File: 09_week_1/removeAt.py
arr = [1, 2, 2.5, 3, 4]

def removeAt(arr, idx):
	removed_val = arr[idx]
	if idx < 0 or idx >= len(arr):
		return None
	for i in range(idx, len(arr)-1, 1):
		arr[i] = arr[i+1]
	arr.pop()	# remove the last value in the list
	return removed_val

print(removeAt(arr, 2))
print(arr)

File: 09_week_1/removeAt.js
var arr = [1, 2, 2.5, 3, 4];

function removeAt(arr, idx) {
	var removed_val = arr[idx];
	if (idx < 0 || idx >= arr.length) {
		return null;
	}
	for (var i = idx, j = arr.length-1; i < j; i++) {
		arr[i] = arr[i+1];
	}
	arr.pop(); // remove the last value
	return removed_val;
}
console.log(removeAt(arr, 2));
console.log(arr);
